docs/userguide/networking/get-started-overlay.md

```diff
@@ -0,0 +1,247 @@
+
+	Because you are in the Swarm master environment, you see all the networks on all Swarm agents. Notice that each `NETWORK ID` is unique.  The default networks on each engine and the single overlay network.  
+
+5. Switch to each Swarm agent in turn and list the network.
+
+			$ eval $(docker-machine env mhs-demo0)
+
+			$ docker network ls
+			NETWORK ID          NAME                DRIVER
+			6b07d0be843f        my-net              overlay             
+			dd51763e6dd2        bridge              bridge              
+			b4234109bd9b        none                null                
+			1aeead6dd890        host                host                
+
+			$ eval $(docker-machine env mhs-demo1)
+
+			$ docker network ls
+			NETWORK ID          NAME                DRIVER
+			d0bb78cbe7bd        bridge              bridge              
+			1c0eb8f69ebb        none                null                
+			412c2496d0eb        host                host                
+			6b07d0be843f        my-net              overlay        
+
+  Both agents reports it has the `my-net `network with the `6b07d0be843f` id.  You have a multi-host container network running!
+
+##  Run an application on your Network
+
+Once your network is created, you can start a container on any of the hosts and it automatically is part of the network.
+
+1. Point your environment to your `mhs-demo0` instance.
+
+			$ eval $(docker-machine env mhs-demo0)
+
+2. Start an Nginx server on `mhs-demo0`.
+
+		$ docker run -itd --name=web --net=my-net --env="constraint:node==mhs-demo0" nginx
+
+	This command starts a web server on the Swarm master.
+
+3. Point your Machin environment to `mhs-demo1`
+
+		$ eval $(docker-machine env mhs-demo1)
+
+2. Run a Busybox instanceand get the contents of the Ngnix server's home page.
+
+		$ docker run -it --rm --net=my-net --env="constraint:node==mhs-demo1" busybox wget -O- http://web
+		Unable to find image 'busybox:latest' locally
+		latest: Pulling from library/busybox
+		ab2b8a86ca6c: Pull complete
+		2c5ac3f849df: Pull complete
+		Digest: sha256:5551dbdfc48d66734d0f01cafee0952cb6e8eeecd1e2492240bf2fd9640c2279
+		Status: Downloaded newer image for busybox:latest
+		Connecting to web (10.0.0.2:80)
+		<!DOCTYPE html>
+		<html>
+		<head>
+		<title>Welcome to nginx!</title>
+		<style>
+		body {
+				width: 35em;
+				margin: 0 auto;
+				font-family: Tahoma, Verdana, Arial, sans-serif;
+		}
+		</style>
+		</head>
+		<body>
+		<h1>Welcome to nginx!</h1>
+		<p>If you see this page, the nginx web server is successfully installed and
+		working. Further configuration is required.</p>
+
+		<p>For online documentation and support please refer to
+		<a href="http://nginx.org/">nginx.org</a>.<br/>
+		Commercial support is available at
+		<a href="http://nginx.com/">nginx.com</a>.</p>
+
+		<p><em>Thank you for using nginx.</em></p>
+		</body>
+		</html>
+		-                    100% |*******************************|   612   0:00:00 ETA
+
+
```

Comment:
This is good info. thanks @moxiegirl.

I would suggest adding 1 more very important information in this document. `External Connectivity` for overlay driver.
I will write something and pass it along to you so that you can modify it to be user-friendly.